Server/mods/deathmatch/logic/luadefs/CLuaWaterDefs.cpp

```diff
@@ -167,11 +169,49 @@ int CLuaWaterDefs::SetWaterLevel(lua_State* luaVM)
     return 1;
 }
 
-int CLuaWaterDefs::ResetWaterLevel(lua_State* luaVM)
+bool CLuaWaterDefs::ResetWaterLevel(std::variant<std::monostate, bool, std::vector<CWater*>, CWater*> resetElements)
 {
     CStaticFunctionDefinitions::ResetWorldWaterLevel();
```

Comment:
Is this function supposed to reset the world water level even if you just want to reset the level for a specific water element?